### PR TITLE
Add react-router topic

### PR DESCRIPTION
**Pull request by sidx8**
### Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: <https://github.com/github/explore/blob/master/CONTRIBUTING.md>.
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?
  - [x] Curating a new topic or collection
---

### Curating a new topic or collection

- [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: <https://github.com/github/explore/tree/master/docs>

> react-router is a popular react tool that has 11509 repositories with that topic, many of which are highly active.

### CODE REVIEW

**Comment by Luois45** [comment]
@sidx8 The logo should have the dimensions of 288x288 pixels.

**Comment by sidx8** [comment]
@Luois45 I have updated the image can you review this

**Comment by Luois45** [comment]
@sidx8 I can't review it because I'm not the code owner but it does look fine now. 👍

**Review by sidx8 — COMMENTED** [review]
created the react-router topic and added a logo

File: topics/react-router/index.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+aliases: react-router, react router
```
Comment (kenyonj) [inline topics/react-router/index.md:2]: This list cannot include the topic name or phrases
```suggestion
```

File: topics/react-router/index.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+created_by: Ryan Florence & Michael Jackson
+display_name: react-router
```
Comment (kenyonj) [inline topics/react-router/index.md:4]: The display name is usually the way the name looks in contexts where it's being written about as a proper noun.
```suggestion
display_name: React Router
```

**Review by kenyonj — CHANGES_REQUESTED** [review]
Some small changes and then we can get this merged!

**Review by kenyonj — APPROVED** [review]
💯 🍨 Thank you for your contribution! I'll merge this in as soon as CI is 🍏

**Pull request merged**